stl/inc/xutility

```diff
@@ -2005,6 +2004,21 @@ _NODISCARD constexpr size_t size(const _Ty (&)[_Size]) noexcept {
     return _Size;
 }
 
+#if _HAS_CXX20
+// FUNCTION TEMPLATES ssize

```

Comment:
```suggestion
// FUNCTION TEMPLATE ssize
```

Reply:
Our convention is to say `FUNCTION TEMPLATE`, singular, for an overload set of a single name.

Reply:
Done